tools/container/runtime/containerd/containerd.go

```diff
@@ -164,3 +169,13 @@ func (o *Options) runtimeConfigOverride() (map[string]interface{}, error) {
 
 	return runtimeOptions, nil
 }
+
+// getContainerdConfigCommand returns a string slice which contains the CLI args to retrieve the current runtime configuration
+func getContainerdConfigCommand(hostRoot string) []string {
```

Comment:
Why is this not defined in the `containerd` config package. It seems as if this logic should be included there?

Reply:
Done